src/JsonSchema/Validator.php

```diff
@@ -26,6 +26,7 @@ class Validator extends BaseConstraint
     const SCHEMA_MEDIA_TYPE = 'application/schema+json';
 
     const ERROR_NONE                    = 0x00000000;
+    const ERROR_ALL                     = ~self::ERROR_NONE;
```

Comment:
Better than using a single bit IMO, because it still works if you use it as a mask :-).